src/App.test.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+
+import { render, fireEvent } from '@testing-library/react';
+
+import App from './App';
+
+describe('App', () => {
+  function renderApp() {
+    return render(<App />);
+  }
+
+  it('초기 상태 확인. input, 할일목록, 완료버튼을 모두 표시한다.', () => {
+    const {
+      container,
+      getByLabelText,
+      getByPlaceholderText,
+      getByText,
+    } = renderApp({
+    });
```

Comment:
이 부분은 이렇게만 처리해도 될 것 같네요~
```suggestion
    } = renderApp();
```